doc/script_commands.txt

```diff
@@ -10589,6 +10601,14 @@ returns progress on success and false on failure
 
 ---------------------------------------
 
+*achievement_iscompleted(<ach_id>)
```

Comment:
I'd like to have the optional parameter `<account ID>`, like `achievement_progress()` has.